### PR TITLE
Program to find letter with highest frequency in a string in Java,C++,Python.

### PR DESCRIPTION
**Pull request by riyaa060**
### Description
I have added a markdown document which find letter with highest frequency in string.

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/basic-dsa/Strings/Highest freqyuency letter.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+- Imports:
+
+##### import java.util.HashMap: This imports the HashMap class, which is used to store letter frequencies.
+
+#### Finding the Highest Frequency Letter:
+
+- char highestLetter = ' '; : This initializes highestLetter to an empty character.
+- int highestCount = 0; : This initializes highestCount to zero.
+- for (char letter : frequency.keySet()) { ... }: This loop iterates over the keys (letters) in the frequency map.
```
Comment (ajay-dhangar) [inline docs/basic-dsa/Strings/Highest freqyuency letter.md]: replace `(char letter : frequency.keySet()) { ... }` with ``` `(char letter : frequency.keySet()) { ... }` ```

File: docs/basic-dsa/Strings/Highest freqyuency letter.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+- if (frequency.get(letter) > highestCount) { ... }: Inside the loop, this condition checks if the current letter’s frequency is greater than highestCount. If so, it updates highestCount and sets highestLetter to the current letter.
```
Comment (ajay-dhangar) [inline docs/basic-dsa/Strings/Highest freqyuency letter.md]: replace `(frequency.get(letter) > highestCount) { ... }` with ``` `(frequency.get(letter) > highestCount) { ... }` ```

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
update your code based on suggestions and can't edit `docs/basic-dsa/Strings/Reversing a string.md` file because this file already merged

**Review by ajay-dhangar — APPROVED** [review]
Done

**Comment by riyaa060** [comment]
Could you please add **hacktoberfest2024** and **hacktoberfest-accepted** labels too? @ajay-dhangar

**Pull request merged**